### DISCUSSION TITLE
app group nosql file descriptors utilization - Not much details

### DISCUSSION THREAD

**Discussion by sirdatta** [netdata/netdata#17169] [Q&A] [answered]
We do not have any nosql instance running. It does not identify which service is causing the issue.

**Comment by ilyam8** [comment, marked as answer]
Hi. [Apps plugin](https://github.com/netdata/netdata/tree/master/src/collectors/apps.plugin#application-monitoring-appsplugin) aggregates processes into groups (charts) and [nosql](https://github.com/netdata/netdata/blob/f307d7a6af08e30e9cadc696a490cd147b9816e7/src/collectors/apps.plugin/apps_groups.conf#L121) is one of them. Metrics for each process (PID) can be found in Top->Processes. Did you check it? If FD % is missing in Processes we can add it.

**Comment by sirdatta** [comment]
FD % is missing. That made it really confusing. Thanks BTW for the response.

We are not using any nosql instances. This is triggering lot of upheaval :)

**Reply by ilyam8** [reply]
> FD % is missing.

There are [FDsLimitPercent and FDs](https://github.com/netdata/netdata/assets/22274335/546b690b-5193-4b7b-be7d-572e569b55f3).
